File: C#/백준/Bronze/13684. Divisão da Nlogõnia/Divisão da Nlogõnia.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            while(true){
                int t = int.Parse(Console.ReadLine());
                if (t==0){
                    break;
                }
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int a = aa[0]; int b = aa[1];
                for (int i=0;i<t;i++){
                    int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                    int c = bb[0]; int d = bb[1];
                    if (c==a || d==b){
                        Console.WriteLine("divisa");
                    }
                    else if(c>a && d>b){
                        Console.WriteLine("NE");
                    }
                    else if(c<a && d>b){
                        Console.WriteLine("NO");
                    }
                    else if(c<a && d<b){
                        Console.WriteLine("SO");
                    }
                    else{
                        Console.WriteLine("SE");
                    }
                }
            }
        }
    }
}

File: Python/백준/Bronze/13684. Divisão da Nlogõnia/Divisão da Nlogõnia.py
while(True):
    n = int(input())
    if n==0:
        break
    a,b = map(int,input().split())
    for i in range(n):
        c,d =  map(int,input().split())
        if c==a or d==b:
            print('divisa')
        elif c>a and d>b:
            print('NE')
        elif c<a and d>b:
            print('NO')
        elif c<a and d<b:
            print('SO')
        elif c>a and d<b:
            print('SE')